site/assets/figures/STYLE.md

```
@@ -0,0 +1,307 @@
+| `2 4` | Initial / light dashes |
+| `3 3` | Standard dashes (grid markers) |
+| `4 3` | Gradient descent / step paths |
+| `4 4` | Residual / trail arcs |
+
+---
+
+## 4. Spacing & Layout
+
+### Interactive Figure Container
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)